ajax/libs/highland/package.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "highland",
+  "version": "2.10.5",
+  "filename": "highland.min.js",
+  "description": "The high-level streams library for Node.js and the browser",
+  "homepage": "http://highlandjs.org/",
+  "keywords": [
+    "streams",
+    "async",
+    "events",
+    "pipe"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/caolan/highland"
+  },
+  "license": "Apache-2.0",
+  "author": "Caolan McMahon <@caolan> (http://caolan.org/)",
```

Comment:
@sufuf3 True. However, `http://caolan.org` automatically points to the `https` version. Need to change it in package.json?